Dalamud/Game/ClientState/Actors/Types/Actor.cs

```diff
@@ -71,6 +72,11 @@ public class Actor : IEquatable<Actor> {
         /// </summary>
         public virtual int TargetActorID => 0;
 
+        /// <summary>
+        ///  Status Effects
+        /// </summary>
+        public StatusEffect[] statusEffects => this.actorStruct.UIStatusEffects;
```

Comment:
Could you fix the style on this? Should be StatusEffects. Thanks!

Reply:
Taken care of